packages/moda-sdk/src/profile/getProfiles.ts

```diff
@@ -0,0 +1,32 @@
+import { isValidAddress } from '../utils/isValidAddress';
+import { ProfileABI } from './abi/ProfileABI';
+import { GetProfileArgs } from './types';
+
+export async function getProfiles(
+  getProfileArgs: GetProfileArgs,
+): Promise<string[]> {
+  const { config, contractAddress, userAddresses } = getProfileArgs;
+  const { publicClient } = config;
+
+  if (publicClient === undefined) {
+    throw new Error('Public client is undefined');
```

Comment:
`Result` 

We are starting to see much of the same errors so maybe we define the error enum inside of `core` as `ConfigError`

```ts
export enum ConfigError {
  ClientUndefined = '...',
  // etc
}
```

Then if any of the functions may have these errors and others you can define the return type with:

```ts
Result<string[], ConfigError | SomeCustomErrorEnum>
```